pkg/module/go_get_fetcher.go

```diff
@@ -120,6 +112,25 @@ func getSources(goBinaryName string, fs afero.Fs, gopath, repoRoot, module, vers
 	return nil
 }
 
+func prepareEnv(gopath string) []string {
+	pathEnv := fmt.Sprintf("PATH=%s", os.Getenv("PATH"))
+	gopathEnv := fmt.Sprintf("GOPATH=%s", gopath)
+	cacheEnv := fmt.Sprintf("GOCACHE=%s", filepath.Join(gopath, "cache"))
+	disableCgo := "CGO_ENABLED=0"
+	enableGoModules := "GO111MODULE=on"
+	cmdEnv := []string{pathEnv, gopathEnv, cacheEnv, disableCgo, enableGoModules}
+	// add Windows specific ENV VARS
+	// if this one is missing we will get an error from go
+	if val, ok := os.LookupEnv("USERPROFILE"); ok {
+		cmdEnv = append(cmdEnv, fmt.Sprintf("USERPROFILE=%s", val))
+	}
+	// if this one is missing go won't give any error but the pkg/mod dir will be empty
```

Comment:
let end up with an error then, with empty packages proxy is not fulfilling its purpose